### PR TITLE
Don't use __name__ for tracers in python docs

### PR DESCRIPTION
**Pull request by cartermp**
Fixes https://github.com/open-telemetry/opentelemetry.io/issues/2517

### CODE REVIEW

**Review by svrnm — APPROVED** [review]
LGTM

File: content/en/docs/instrumentation/python/getting-started.md [hunk 1]
```diff
@@ -376,9 +376,9 @@ from opentelemetry import metrics
 from random import randint
 from flask import Flask, request
 
-tracer = trace.get_tracer(__name__)
+tracer = trace.get_tracer("diceroller.traces")
```
Comment (JamieDanielson) [inline content/en/docs/instrumentation/python/getting-started.md:379]:
```suggestion
tracer = trace.get_tracer("diceroller.tracer")
```

File: content/en/docs/instrumentation/python/getting-started.md [hunk 2]
```diff
@@ -376,9 +376,9 @@ from opentelemetry import metrics
...
 # Acquire a meter.
-meter = metrics.get_meter(__name__)
+meter = metrics.get_meter(diceroller.metrics)
```
Comment (JamieDanielson) [inline content/en/docs/instrumentation/python/getting-started.md:381]:
```suggestion
meter = metrics.get_meter(diceroller.meter)
```

File: content/en/docs/instrumentation/python/getting-started.md [hunk 3]
```diff
@@ -251,7 +251,7 @@ from random import randint
 from flask import Flask, request
 
 # Acquire a tracer
-tracer = trace.get_tracer(__name__)
+tracer = trace.get_tracer("diceroller.traces")
```
Comment (JamieDanielson) [inline content/en/docs/instrumentation/python/getting-started.md:254]:
```suggestion
tracer = trace.get_tracer("diceroller.tracer")
```

**Review by JamieDanielson — APPROVED** [review]
nit: Think it would be more consistent to have "tracer" and "meter" instead of "traces" and "metrics" in the names. Non-blocking, the overall change is necessary and good 👍

**Comment by cartermp** [comment]
@open-telemetry/python-approvers can one of y'all take a look?

**Review by mhausenblas — APPROVED** [review]

**Review by chalin — COMMENTED** [review]
Pinging @open-telemetry/python-approvers.

Also note that this PR will need to be rebased and adjusted following #2553, which uses `__name__`.

**Comment by chalin** [comment]
Final ping for @open-telemetry/python-approvers

**Pull request merged**